subscription_oca/models/sale_order.py

```diff
@@ -49,7 +49,7 @@ def create_subscription(self, lines, subscription_tmpl):
             rec = self.env["sale.subscription"].create(
                 {
                     "partner_id": self.partner_id.id,
-                    "user_id": self._context["uid"],
+                    "user_id": self._context.get("uid", self.env.uid),
```

Comment:
Oops didn't see this:

`self.env.context.get(...)`is better as _context is deprecated since a long time